docs/reference/remote-build.rst

```diff
@@ -0,0 +1,56 @@
+.. _ref-remote-build:
+
+
+*************
+Remote builds
```

Comment:
@medubelko in the end I opted to document this in Rockcraft now, so that the release has the docs, and then worry about upstreaming this to craft-application in the near future. Once we review and merge this I can point to this new doc on the 1.7.0 release notes